src/data/faq.json

```diff
@@ -135,10 +135,10 @@
                         "anchor": "vac_cert_invalid",
                         "active": true,
                         "textblock": [
-                            "There are currently (as of 9 June 2021) no vaccination certificates with QR codes that can be scanned with the app.",
-                            "Despite the QR code, any document issued in the past and currently is not a valid digital vaccination certificate, but a vaccination proof that follows the German infection protection act, but not the new coordinated European guidelines for a digital vaccination certificate.",
-                            "These new digital vaccination certificates will be issued soon. Such digital certificates can only be issued by authorized bodies via the central interface developed by other partners on behalf of the federal government. The field test is now completed, and the function is now being rolled out nationwide.", 
-                            "It has not yet been conclusively clarified who exactly, in addition to the vaccination centers, will be able to issue a vaccination certificate on the basis of proof of vaccination, i.e. the yellow vaccination booklet or the above-mentioned vaccination proof. The network will be gradually expanded."
+                            "There are currently (as of 10 June 2021) only a few places from which certificates with QR codes are issued, which can be scanned with the app.",
+                            "Usually, any document until today is not yet a valid digital vaccination certificate, but just a vaccination documentation according to §22 of the German Infection Protection Act, that is incompatible to the latest aligned European guidelines for a digital vaccination certificate. Therefore, this document, as well as the barcodes and QR codes in the yellow vaccination booklet, <b>cannot</b> be scanned with the Corona-Warn-App.",

```

Comment:
"incompatible to" =>
"incompatible with"
https://dictionary.cambridge.org/dictionary/english/incompatible

Reply:
thanks.

Reply:
It still says "incompatible to", not "with".
https://lingohelp.me/preposition-after-adjective/incompatible-with-for-in-to/

I think that "with" is better.